db/schema.rb

```diff
@@ -11,11 +11,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20151024110515) do
+ActiveRecord::Schema.define(version: 20151024161110) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "books", force: :cascade do |t|
+    t.string   "title",      null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  add_index "books", ["title"], name: "index_books_on_title", using: :btree
```

Comment:
I added an index since we'll be sorting on this shortly. I could have waited until we needed that behaviour, but I find it's easy to forget to add an index it you don't do it at the same time as creating the table.